AdyenDropIn/DropInComponent.swift

```diff
@@ -212,9 +214,7 @@ extension DropInComponent: PaymentComponentDelegate {
     
     /// :nodoc:
     public func didFail(with error: Error, from component: PaymentComponent) {
-        paymentInProgress = false
```

Comment:
we don't need to set it to `false` here?

Reply:
It is set in `userCancel` now